### PR TITLE
Add a “Become a Teacher” form on Get Involved page

### PR DESCRIPTION
**Pull request by crayolakat**

### CODE REVIEW

File: src/views/InvolvementPanel/InvolvementPanel.js [hunk 1]
```diff
@@ -18,6 +19,10 @@ export default class InvolvementPanel extends React.Component {
         text: 'Volunteer',
         view: <VolunteerFormView />
       },
+      TEACHER: {
```
Comment (bibekg) [inline src/views/InvolvementPanel/InvolvementPanel.js:22]: Now that we have 4 form links on the switcher bar, how does it look as you reduce the width of the window? Does the text wrap and look odd?

Comment (crayolakat) [reply src/views/InvolvementPanel/InvolvementPanel.js:22]: It reduces fine in Google Chrome and Microsoft Edge. Are you aware that the text of the Get Involved page does not display properly in Internet Explorer? Not just in this branch but on the current website right now.
screenshot 13

Comment (bibekg) [reply src/views/InvolvementPanel/InvolvementPanel.js:22]: Oh, interesting. I wasn't aware. I unfortunately don't have a platform with which to test Internet Explorer. Would you mind debugging and figuring out what's causing that?

File: src/views/InvolvementPanel/FormViews/TeacherFormView.js [hunk 2]
```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+import FormView from './FormView';
+
+export default class TeacherFormView extends React.Component {
+ render() {
+
+  const formMessage = "Join our teaching team and change a child’s life. We have a wide range of teaching opportunities available, from going into classrooms in cities across the country to teaching from the comfort of your own home.";
+  const inputFields = [
```
Comment (bibekg) [inline src/views/InvolvementPanel/FormViews/TeacherFormView.js:9]: Can you reach out to Lisa/Kiera and confirm that these are the fields we want on this form?

**Review by bibekg — APPROVED** [review]
Everything looks good. Just confirm form fields with Lisa/Kiera since they didn't explicitly say which ones we want here.

**Pull request merged**